content/millennium/r4/financial/general/charge-item.md

```diff
@@ -212,6 +212,61 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Modify
+
+<%= beta_tag(action: true) %>
+
+Modify a ChargeItem
```

Comment:
Let's update this description to something like "Modifies an existing ChargeItem resulting in a newly created ChargeItem."

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/851312df1b650432b1463c74d6aa86cfe9d025ec#diff-03f941e4ceb2839bfe0b49a5cc913e04919560e289603ae7b08b875f86ec568dR219